packages/browser/src/transports/base.ts

```diff
@@ -34,10 +35,20 @@ export abstract class BaseTransport implements Transport {
   /** Locks transport after receiving rate limits in a response */
   protected readonly _rateLimits: Record<string, Date> = {};
 
+  protected _outcomes: { [key: string]: number } = {};
```

Comment:
```suggestion
  protected _outcomes: Partial<Record<Outcome, number>> = {};
```

Reply:
Can also do
```suggestion
  protected _outcomes: { [key in Outcome]?: number} = {};
```

Reply:
The thing is that its not an Outcome. Its `category:outcome` as a key.

Reply:
Ah my bad, still kind of confusing though. Can we add a comment that explains what the key is supposed to be?